### ISSUE TITLE
Investigate sphinx-autoapi warning

### ISSUE THREAD

**Issue by astrojuanlu** [poliastro/poliastro#1077]
We keep getting this warning:

```
/home/docs/checkouts/readthedocs.org/user_builds/poliastro/envs/1073/lib/python3.7/site-packages/autoapi/extension.py:137: RemovedInAutoAPI2Warning: autoapi_template_dir will be expected to be  relative to conf.py instead of relative to where sphinx-build is run
```

**Comment by DhruvSondhi** [comment]
This warning is evident as the **Search** functionality of the documentation doesn't work. I am trying to find out how we can overcome this & will update the same :smile:

**Comment by DhruvSondhi** [comment]
This has been fixed via #1090

**Comment by astrojuanlu** [comment]
Indeed!
